docs/integrations/javascript/nextjs.md

```diff
@@ -0,0 +1,56 @@
+---
+integration: logfire
+---
+
+# Next.js
```

Comment:
again, maybe we have to mention compatibility (like Next.js 13+ with App Router). I remember had some pain with different versions of Next.js